app/views/placements/support/providers/users/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= content_for :page_title, sanitize(@provider.name) %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-l"><%= @provider.name %></h1>
+      <%= render SecondaryNavigationComponent.new do |component| %>
+        <% component.with_navigation_item t(".secondary_navigation.details"), "#" %>
+        <% component.with_navigation_item t(".secondary_navigation.users"), placements_support_provider_users_path(@provider) %>
+        <% component.with_navigation_item t(".secondary_navigation.mentors"), "#" %>
+        <% component.with_navigation_item t(".secondary_navigation.placements"), "#" %>
+        <% component.with_navigation_item t(".secondary_navigation.providers"), "#" %>
+      <% end %>
+    </div>
+
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m"><%= t(".heading") %></h2>
+      <%= govuk_button_to(t(".add_user"), new_placements_support_provider_user_path(@provider), method: :get) %>
+
+      <table class="govuk-table">
```

Comment:
There is also a `govuk_table` helper, here is an example of it in use:

https://github.com/DFE-Digital/register-trainee-teachers/blob/bd71de101065fc2cd9825295ca8053b55b0d236e/app/views/guidance/dates_and_deadlines.html.erb#L13

Reply:
resolved in [853411c](https://github.com/DFE-Digital/itt-mentor-services/pull/125/commits/853411c3dc199496f425ee6adeb80fc5690ef163)